docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md

```diff
@@ -0,0 +1,118 @@
+# 使用Helm部署Nebula Graph集群
+
+## 前提条件
+
+[安装Nebula Operator](2.deploy-nebula-operator.md)
+
+## 创建Nebula Graph集群
+
+1. 添加Nebula Operator chart至Helm仓库（如已创添加，请略过1至2步）。
+   
+	```bash
+	helm repo add nebula-operator https://vesoft-inc.github.io/nebula-operator/charts
+	```
+
+2. 更新Helm仓库，拉取最新仓库资源。
+   
+	```bash
+	helm repo update
+	```
+
+3. 配置Helm的环境变量。
+	```bash
+	export NEBULA_CLUSTER_NAME=nebula         # Nebula Graph集群的名字。
+	export NEBULA_CLUSTER_NAMESPACE=nebula    # Nebula Graph集群所处的命名空间的名字。
+	export STORAGE_CLASS_NAME=gp2             # Nebula Graph集群的StorageClass。
+	```
+
+4. 为Nebula Graph集群创建命名空间（如已创建，请忽略此步）。
+
+	```bash
+	kubectl create namespace "${NEBULA_CLUSTER_NAMESPACE}"
+	```
+
+5. 创建Nebula Graph集群。
+
+	```bash
+	helm install "${NEBULA_CLUSTER_NAME}" nebula-operator/nebula-cluster \
+			--namespace="${NEBULA_CLUSTER_NAMESPACE}" \
+			--set nameOverride=${NEBULA_CLUSTER_NAME} \
+			--set nebula.storageClassName="${STORAGE_CLASS_NAME}"
+	```
+
+6. 查看Nebula Graph集群创建状态。
+   
+	```bash
+	kubectl -n "${NEBULA_CLUSTER_NAMESPACE}" get pod -l "app.kubernetes.io/cluster=${NEBULA_CLUSTER_NAME}"
+	```
+  返回示例：
+	```bash
+	NAME                READY   STATUS    RESTARTS   AGE
+	nebula-graphd-0     1/1     Running   0          5m34s
+	nebula-graphd-1     1/1     Running   0          5m34s
+	nebula-metad-0      1/1     Running   0          5m34s
+	nebula-metad-1      1/1     Running   0          5m34s
+	nebula-metad-2      1/1     Running   0          5m34s
+	nebula-storaged-0   1/1     Running   0          5m34s
+	nebula-storaged-1   1/1     Running   0          5m34s
+	nebula-storaged-2   1/1     Running   0          5m34s
+	```
+
+## 扩缩容集群
+
+用户可通过定义Nebula Graph中不同服务对应的`replicas`的值扩缩容Nebuala Graph集群。
+
+例如：扩容Nebula Graph集群中Storage的副本数为5（原始值为2），命令如下：
+
+```bash
+helm upgrade "${NEBULA_CLUSTER_NAME}" nebula-operator/nebula-cluster \
+    --namespace="${NEBULA_CLUSTER_NAMESPACE}" \
+    --set nameOverride=${NEBULA_CLUSTER_NAME} \
+    --set nebula.storageClassName="${STORAGE_CLASS_NAME}" \
+    --set nebula.storaged.replicas=5
+```
+同理，将Nebula Graph集群中服务对应的`replicas`的值设置低于原始值，即可实现集群服务的缩容。
+
+!!! caution
+
+		目前仅支持对Nebula Graph集群中的Graph服务和Storage服务进行扩缩容，不支持扩缩容Meta服务。
+
+用户可点击[nebula-cluster/values.yaml](https://github.com/vesoft-inc/nebula-operator/blob/master/charts/nebula-cluster/values.yaml)查看Nebula Cluster集群chart的更多配置。有关文件中配置项解释，参考下文**附录：Nebula Graph集群chart配置参数说明**。
```

Comment:
use a relative path instead of the absolute path. so the it can work for multi-verson